src/snowflake/snowpark/column.py

```diff
@@ -990,12 +994,14 @@ def _create_ast(
         ast = proto.Expr()
         if property is not None:
             prop_ast = getattr(ast, property)
+            fill_src_position(prop_ast.src)
             for attr, value in assign_fields.items():
                 setattr_if_not_none(prop_ast, attr, value)
             for attr, value in assign_opt_fields.items():
                 setattr_if_not_none(getattr(prop_ast, attr), "value", value)
             for attr, messg in copy_messages.items():
```

Comment:
The standard abbreviation for "message" is "msg". "Messg" is so long, we might as well fully spell it out. I'd prefer `msg`.